service/src/main/java/org/cbioportal/genome_nexus/service/internal/VEPEnrichmentService.java

```diff
@@ -68,7 +68,8 @@ public void registerEnricher(String id, AnnotationEnricher enricher)
         // initiate enricher list if not initiated yet
         if (enrichers == null)
         {
-            enrichers = new HashMap<>();
+            // linked hash map ensures that enrichers are executed by the same order they are registered
+            enrichers = new LinkedHashMap<>();
```

Comment:
This is important to make sure that certain enrichers are always executed first. For example, we want the isoform override enricher to run first so that we pick the correct canonical transcript.